annoying/fields.py

```diff
@@ -127,7 +127,9 @@ def get_db_prep_save(self, value, *args, **kwargs):
             return None
         if isinstance(value, dict) or isinstance(value, list):
             value = json.dumps(value, cls=DjangoJSONEncoder)
-        return super(JSONField, self).get_db_prep_save(value, *args, **kwargs)
+        else:
+            return super(JSONField, self).get_db_prep_save(value, *args, **kwargs)
+        return value
```

Comment:
Can you put the return in the `if` above, for more clarity, since it's just an if/else? I was kind of stumped for a minute, wondering how execution would ever reach this.